README.md

```diff
@@ -54,17 +54,19 @@ To start the app, run these commands, replacing `$(pwd)/workspace` with the dire
 ```bash
 # The directory you want OpenDevin to work with. MUST be an absolute path!
 export WORKSPACE_BASE=$(pwd)/workspace;
+export SSH_PASSWORD="set some long password here";
 ```
 
-> [!WARNING]  
-> OpenDevin runs bash commands within a Docker sandbox, so it should not affect your machine. 
+> [!WARNING]
+> OpenDevin runs bash commands within a Docker sandbox, so it should not affect your machine.
 > But your workspace directory will be attached to that sandbox, and files in the directory may be modified or deleted.
 
 ```bash
 docker run \
```

Comment:
There's another `docker run` command in the docs folder too--we'll need to change there as well